ajax/libs/reel.js/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "reel.js",
+  "filename": "jquery.reel-min.js",
+  "version": "1.3.0",
+  "description": "Reel is an established and the most versatile three-sixty player for jQuery",
```

Comment:
@kennynaoh I think you can reference [reel.jquery.json](https://github.com/pisi/Reel/blob/v1.3.0/reel.jquery.json). You can modify `description`, `repository`, `keywords`, `author` as same as upstream.